ci/azure-cross-compile.yml

```diff
@@ -7,18 +7,14 @@ jobs:
     strategy:
       matrix:
         iOS_64:
-          vmImage: macOS-10.13
+          vmImage: macOS-10.15
```

Comment:
Any reason this is `10.15` and not `10.14`? I think we should keep this in line with [Tokio's versions](https://github.com/djc/tokio/blob/cadbedc3d633f24c069e4a8ef36108148d145ec7/ci/azure-check-features.yml#L8-L9)

Reply:
No reason, 10.15 is just latest so we don't have to update next year. If Tokio tests on 10.14 and we test on 10.15 then we got both covered. But I don't really have a strong preference either way, so if you want me to change it I'm ok with it.

Reply:
I'm okay with that.